.github/workflows/test-e2e-flow.yml

```diff
@@ -0,0 +1,62 @@
+name: Test composite actions on SBOM+SLSA example
+on:
+  push:
+    branch:
+      - main
+    paths:
+      - "scai-gen/**"
+  # Want to trigger these tests whenever the Go CLI or
+  # APIs are modified
+  pull_request:
+    paths:
+      - "scai-gen/**"
+
+jobs:
+  sbom-slsa-ex:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe
+        with:
+          go-version: 1.20.x
+
+      - name: Checkout updated scai-gen CLI tools
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+
+      - name: Setup Env
+        run: |
+          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
+
+      - name: Install scai-gen CLI tools
+        shell: bash
+        run: |
+          go install ./scai-gen
+          mkdir -p temp
+
+      - name: Generate SBOM SCAI AttributeAssertion
+        id: gen-sbom-assert
+        uses: marcelamelara/scai-demos/.github/actions/scai-gen-assert@add-scai-reusable-workflows
```

Comment:
hm, do these get bumped up in a separate PR?

Reply:
I wonder if you could `uses: ./.github/actions` etc. https://github.com/gittuf/gittuf-installer/blob/main/.github/workflows/test.yml#L29

I'm not sure though.

Reply:
I've been trying to figure this out, too. I'll try your suggestion with relative paths, thanks!